README.md

```diff
@@ -28,6 +28,8 @@ The external scaler calls Cosmos DB APIs to estimate the amount of changes pendi
 
 ## Setup Instructions
 
+> :warning: **Caution:** The scaler uses [.NET SDK Version 3](https://github.com/Azure/azure-cosmos-dotnet-v3) client library to estimate pending changes inside the change feeds. This library uses a different ID-naming scheme than its predecessors when creating and processing lease documents inside the lease container. This makes it incompatible with the deprecated [.NET SDK Version 2](https://github.com/Azure/azure-cosmos-dotnet-v2) and currently available [Java SDK](https://github.com/Azure/azure-cosmosdb-java) libraries. If you used one of the latter libraries to create the change feed processor, the external scaler would to unable to detect pending items and may scale down your application to `minReplicaCount` as defined in the `ScaledObject`.
```

Comment:
Current is a point in time concept, would you mind mentioning the Java SDK versions please?

Reply:
@tomkerkhove, we received clarification on this from Cosmos DB team. Instead of latest .NET SDK, Java SDK is the outlier. Updated the note accordingly.